### ISSUE TITLE
about RateSAN model 

### ISSUE THREAD

**Issue by ding15963** [combra-lab/pop-spiking-deep-rl#2]
Hi Guangzhi，
Thank you very much for sharing your work, I am very interested in it. But the training code of RateSAN model in mujoco is not public, I know it is your IROS work, but our own implementation is not good, I hope you can provide your RateSAN training test code, I hope you can help us, thanks.

**Comment by michaelgzt** [comment]
Hi Ding,

Thanks for your interest in our work. 

You are right, the RateSAN implementation of this work is very similar to our previous IROS work (source code here: https://github.com/combra-lab/spiking-ddpg-mapless-navigation/blob/master/training/train_spiking_ddpg/sddpg_networks.py)

The only differences are the input encoding and output decoding. For both the input layer and output layer, we used two groups of neurons to represent negative and positive values.

For input encoding, we first normalize the observation using the same way as for PopSAN: https://github.com/combra-lab/pop-spiking-deep-rl/blob/5fe5f39eeb859af978bfc52a796e7b8a5ff7517e/popsan_drl/popsan_td3/replay_buffer_norm.py#L92 
Then, we rescale the observation to [-1, 1] using the clip_limit value (we set this value to 3.0):
`norm_obs = norm_obs / self.clip_limit`
The rescaled observation value is encoded in the negative and positive neurons using rate encoding.

For output decoding, we do subtraction on the firing rate between the negative and positive output neurons and get a value within [-1, 1] for control.
